### PR TITLE
Pull request for apple clone page

### PR DESCRIPTION
**Pull request by ioilmio**
I'm making this pull request after installing and using Linux ubuntu for the first time
i have some issue with stylelint so i decided to open a pull request for the project and check if it will run after that.
anyway i know that i have to complete the project but if you have some advice or hint don't hesitate to let me know

### CODE REVIEW

**Comment by ioilmio** [comment]
linter is spitting out an error about font-family referred to FontAwesome used for the search icon

File: style.css
Comment (ioilmio) [inline style.css]: this is the only one line stickler is complaining about

**Comment by lekegitrepo** [comment]
### This PR is under Review by [TSE] _Temitayo Adeleke_

**Comment by lekegitrepo** [comment]
### Hi @ioilmio  👍 
**Good job**. **_I saw your comments and I understood perfectly what you are facing concerning the stickler issue_**. **_However, I can not change the requirements_**. **_I'll suggest you should find a way to implement this in HTML instead of CSS implementation in your style sheet._**
Check the comments below for changes you need to implement before the project can be approved.

1. The four elements at the bottom of the should be links (to other articles or #) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

2. There should margin between the content in the footer, the stack upon each other

**_This is your page:_**

image

**_and this is the original:_**

image

3. Improve your readme file and you can also add live demo link.

4. Try to improve your pull request comment to reflect what you did in the project.

5. Fix this error in the Html file:

image

6. As suggested above find a way to implement the fontawesome icon in HTML: with syntax like this `<i class="fas fa-search"></i>` input-icon class element.

**_Kindly make these changes and resubmit for review._**

if you have any doubt or question you can contact me on slack.

### Reviewed by: _Temitayo Adeleke_

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
### Status: Require Changes

**Comment by ioilmio** [comment]
First thanks for the advice, then i tried to implement various kinds of positioning techniques like grid float and flex,i used icons for navbar and input field, and background image for the main section.
i fixed linter and validators errors refactoring the navbar using div and span instead of list elements.
Waiting for approval i hope.
Thanks for your time.

**Comment by davidmrtz-dev** [comment]
Pull Request under review by David Elí 👨‍💻

**Comment by davidmrtz-dev** [comment]
### **Status: Changes required.** 🔍 ♻️ 
****
### Hi Team. 😄 
### Very good job so far, I notice you do a great job implementing the changes required by the previous TSE 🎊 
### However, the following issues need to be fixed before moving on  👨‍💻  ⬇️ 

****

- [x] Navbar.
**The first eight elements should be links (to other sections or #).**
I notice you use on your navbar the divs required by the previous TSE, but there's only a little change to get this working. You can do this adding a link tag "a" inside each of the spans of your navbar, and style it using CSS to remove the default link color (blue).

****

- [x] Footer.
Add the fonts and colors required to your footer so that it looks as close as possible to the example on which the project is based.
Remember, we are making a clone. 😄 

Your page. ⬇️ 

Screen Shot 2020-03-05 at 10 53 51 PM

The example on which the project is based. ⬇️ 

Screen Shot 2020-03-05 at 10 54 06 PM

You can do this by adding the fonts required on the head of your HTML file.
You can pick the link from google fonts. 

****

### **Please kindly make the changes required and submit a new code review once you are ready!** 🙏 

Happy coding! 😄 💻  
If you have any questions or need help, you can message me on slack (@ David Eli Martinez Garcia).

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Comment by ioilmio** [comment]
Thanks for the advice and your time hope this will satisfied requirements.

**Comment by uryelah** [comment]
Hello **Salvo Illuminato**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**You've been super cooperative with the code reviews fixes, good job! 👍** 
Hang in there because there are still a couple of fixes to make this project even more awesome.

---------    

**Great job at: 🥇** 
- good use of gradients
- using background images
- matching the footer text color

**You still need to fix the issues below before you can move to the next project**

---------

### Required changes 🛠 

- [x] Please don't use ids for styling in your CSS file. [Reasons why here.](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)

- [x] Your pull request description should describe what you made in the project. [Check here for a bullet point example.](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

- [x] The original page has some margin in the borders of the body. Please make sure to add that.

- [x] The original page main text has a lighter font weight than yours, please make it similar to the original one.
Your version:
Screenshot_2020-03-06 Apple Clone(1)

Original version:
Screenshot_2020-03-06 Apple(1)

- [x] Like the TSEs above said the nav links should be `<a>`s. Not just the text of each, the link should be the whole 'button'. [Check the original page, in yours only the text itself is clickable.](https://web.archive.org/web/20140301004610/http://www.apple.com/)
TIP: You can change the display property of the `<a>` to inline-block so it can work like a `<div>`.

---------

### Optional changes

- [ ] [OPTIONAL] Change the tag of the main image to a `<main>` tag, which will make your page [more semantic.](https://dev.to/kenbellows/why-i-care-about-the-semantic-web-2kn7)

- [x] [OPTIONAL] Add a screenshot of your project to your readme file.

    
    
**Please fix the required changes 👷  before asking for a new code review**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by ioilmio** [comment]
- nav buttons are now links to #(homepage)

- used classes instead of ids

- lighter font for hero text(Roboto low weight font)

- margin for body (inside body selector 2px on bottom)

- added screenshot to readme using issues form and copy-paste the link

**Comment by thsvr** [comment]
Project under review by Thaís Vieira (TSE)

**Comment by thsvr** [comment]
### **Status: PROJECT APPROVED** :1st_place_medal: :1st_place_medal: :1st_place_medal: 

Hi @ioilmio :smile: 

You did all the required changes asked by Sarah (TSE) and your  project looks amazing :fireworks: :fireworks: 

So, please submit the project completion form.

:v:  **One suggestion for the next projects:**  you can create a feature branch with a more descriptive name, like ''main-page'' for example. You will use development (or develop) branch on the next sections as your golden branch and if you are curious about it you can read this article [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow )

If you have any questions, don't hesitate to contact me through slack [@Thaís Vieira Azevedo](url)

### **Happy coding** :computer: and good luck on the next project :desktop_computer:  :computer_mouse: 

**Thaís Vieira (TSE)**

**Pull request merged**
